### DISCUSSION TITLE
How to get hover tooltip with polygons?

### DISCUSSION THREAD

**Discussion by sdtaylor** [opengeos/leafmap#658] [Q&A] [answered]
In the choropleth example notebook (https://leafmap.org/notebooks/53_choropleth/) is has a neat default feature with tooltips showing the polygon attributes over the specific polygons. But when I try it myself the attributes are now down in the corner:

![image](https://github.com/opengeos/leafmap/assets/12282062/e79bbb98-e0dc-455b-bb1f-6087d218920f)

Is there a way to have the tooltips back like in the example notebook?

**Comment by giswqs** [comment, marked as answer]
The documentation website uses the folium backend. If you use `import leafmap.foliumap as leafmap` rather than `import leafmap`, it should work. The ipyleaflet backend can only show the tooltip in the lower-right corner.
